cli/utils.js

```diff
@@ -188,7 +188,12 @@ const createYeomanEnv = () => {
     Object.keys(SUB_GENERATORS)
         .filter(command => !SUB_GENERATORS[command].cliOnly)
         .forEach(generator => {
-            env.register(require.resolve(`../generators/${generator}`), `${CLI_NAME}:${generator}`);
+            if (SUB_GENERATORS[generator].blueprint) {
+                /* eslint-disable prettier/prettier */
+                env.register(require.resolve(`../node_modules/${SUB_GENERATORS[generator].blueprint}/generators/${generator}`), `${CLI_NAME}:${generator}`);
```

Comment:
this expects the package to be installed on the node_modules of the generator? Shouldn't you rely on npms resolution mechanism here

Reply:
You are totally right !